roles/ansible-os-hardening/vars/main.yml

```diff
@@ -36,15 +27,12 @@ os_security_suid_sgid_whitelist: []
 os_security_suid_sgid_remove_from_unknown: false
 
 # remove packages with known issues
-os_security_packages_clean: true
+os_security_packages_clean: false
```

Comment:
Oh, I get the confusion. The implementation discusses a default setting to false, but we ultimately decided to keep true default. See https://github.com/hardening-io/chef-os-hardening/pull/12/files#diff-25e5d4a4446ae12a0d6f1162b6160375R81

I add a ticket to the test to make this more explicit, because it is confusing indeed.